sbc_compassion/tools/import_letter_functions.py

```diff
@@ -21,13 +21,117 @@
 import checkboxreader as cbr
 import bluecornerfinder as bcf
 import numpy as np
+from math import ceil
 from glob import glob
 from wand.image import Image
 
+from openerp import _
+
 
 ##########################################################################
 #                           GENERAL METHODS                              #
 ##########################################################################
+def testline(env, line_vals, csv_file, doc_name):
+    """ Test each line and verify the data given by the csv.
+    If a line does not correspond to the csv, this function erases it.
+    :param env: Odoo environment
+    :param dict line_vals: Dictionnary that will be used to create line
+    :param One2many('test.import.letter.line') csv_file: CSV used to check
```

Comment:
This doesn't make sense to me : a param cannot be a One2many. One2many is a field, It can be a Recordset of 'test.import.letter.line' objects. But is it a recordset or a CSV file ?